yt_dlp/extractor/mildom.py

```diff
@@ -287,29 +274,28 @@ class MildomUserVodIE(MildomBaseIE):
             'id': '10093333',
             'title': 'Uploads from ねこばたけ',
         },
-        'playlist_mincount': 351,
+        'playlist_mincount': 732,
```

Comment:
any point in increasing the mincount?

Reply:
To notice bugs in paging in the future (or massive deletion)